lib/oli_web/templates/email/_footer.html.eex

```diff
@@ -1,9 +1,21 @@
+<%
+    company_address = Oli.VendorProperties.company_address()
+    product_learn_more_link = Oli.VendorProperties.product_learn_more_link()
+%>
+
 <table align="center" role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%" style="margin: auto;">
     <tr>
         <td style="padding: 20px; font-family: sans-serif; font-size: 12px; line-height: 15px; text-align: center; color: #212529;">
             <br><br>
             <%= Oli.VendorProperties.product_full_name() %><br>
-            <%= Oli.VendorProperties.company_name() %><br><span class="unstyle-auto-detected-links"><%= Oli.VendorProperties.company_address() %></span>
+            <%= Oli.VendorProperties.company_name() %><br>
+            <%= if company_address != "" do %>
+                <span class="unstyle-auto-detected-links"><%= company_address %></span>
+            <% else %>
+                <a href="<%= product_learn_more_link %>" style="color: #2c66c4">
```

Comment:
Could we move this inline style to the stylesheet? Would also make it better in the future if standard color scheme is adopted.